go-controller/pkg/node/ovspinning/ovspinning_noop.go

```diff
@@ -0,0 +1,18 @@
+//go:build !linux
+// +build !linux
+
+package ovspinning
+
+import (
+	"fmt"
+	"strconv"
+	"time"
+
+	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/util"
+	"golang.org/x/sys/unix"
+	"k8s.io/klog/v2"
+)
+
+func Run(_ <-chan struct{}) {
+	klog.Infof("OVS CPU pinning is supported on linux platform only")
+}
```

Comment:
So does this really get executed when we're not on linux? I don't see how, but I might very well be missing something :)   If it is necessary, the imports above are not used apart from klog.

Reply:
good catch about the imports. 
The lines at the beginning of the file prevent this file to built for linux:

```
//go:build !linux
// +build !linux
```